src/fdk_rdf_parser/parse_functions/concept.py

```diff
@@ -321,4 +321,5 @@ def parse_concept(graph: Graph, fdk_record_uri: URIRef, concept_uri: URIRef) ->
         exactMatch=value_set(graph, concept_uri, SKOS.exactMatch),
         closeMatch=value_set(graph, concept_uri, SKOS.closeMatch),
         memberOf=value_set(graph, concept_uri, uneskos_uri("memberOf")),
+        relationRole=object_value(graph, concept_uri, skosno_uri("relationRole")),
```

Comment:
relasjonsrolle kan også komme som skos:Concept, ikke bare som Literal

ref: https://data.norge.no/specification/skos-ap-no-begrep#Assosiativ-begrepsrelasjon-relasjonsrolle

Reply:
er det tegn på at den trenger sin egen parse funksjon? `extract_relationRole`?

Reply:
feks, eventuelt prøve å lage en mer generisk metode, siden dette ser ut til å være tilfelle for flere